### PR TITLE
PL generation capacity update

### PR DESCRIPTION
**Pull request by wojciej**
I have noticed that live data on entsoe for biomas generation capacity are much higher than generation capacity in entsoe themself.
I dug deeper and I have end up on Polish Energy Regulator Office website where I have found data for Q4 2019 showing that available generation capacity for biomass in poland is actually: 1 492,875MW

https://www.ure.gov.pl/download/9/10905/mocIVkw2019.pdf

I will add this into the resources in readme

Also - I have updated PL solar capacity as per new reading from PSE.

I also have reverted changes introduced in #2412 as per discussion in that thread.

### CODE REVIEW

**Comment by alixunderplatz** [comment]
@wojciej you see larger values for biomass electricity on the map, because since 10th April, solar data is reported within the biomass category to entso-e.
compare with this chart (select only biomass):
http://www.energodock.com/poland/electricity-generation

**Comment by wojciej** [comment]
@alixunderplatz thanks for this explanation - I was not aware of that. Would you be able to put some source of this change on entsoe side please ? I would love to read more abt it.

Nevertheless I would still process with this update as the data from URE seems to be legit.

**Comment by wojciej** [comment]
Hi @jarek as per our discussion I have reverted changes introduced in #2412.
I have also updated PL solar capacity as per newest reading from PSE.

File: README.md [hunk 1]
```diff
@@ -307,8 +307,8 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
 - Panama: [ETESA] (https://www.cnd.com.pa/informes.php?cat=5)
 - Peru: [IRENA](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Mar/IRENA_RE_Capacity_Statistics_2020.pdf)
 - Poland  
-  - Solar: [PSE Twitter](https://twitter.com/pse_pl/status/1248502288152293378)
-  - Other Renewable: [IRENA](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Mar/IRENA_RE_Capacity_Statistics_2020.pdf)
+  - Solar: [PSE Twitter](https://twitter.com/pse_pl/status/1259734669232427009)
+  - Biomas: [URE] (https://www.ure.gov.pl/download/9/10905/mocIVkw2019.pdf)
```
Comment (jarek) [inline README.md:311]:
```suggestion
  - Biomass: [URE] (https://www.ure.gov.pl/download/9/10905/mocIVkw2019.pdf)
```

minor spelling

**Pull request merged**